recipes/aws-noaa-whoi/meta.yaml

```
@@ -0,0 +1,26 @@
+title: 'AWS NOAA WHOI SST'
+description: 'Analysis-ready datasets derived from AWS NOAA WHOI NetCDF'
+pangeo_forge_version: '0.9.2'
+pangeo_notebook_version: '2021.07.17'
+recipes:
+  - id: aws-noaa-sea-surface-temp-whoi
+    object: 'recipe:recipe'
+provenance:
+  providers:
+    - name: 'AWS NOAA Oceanic CDR'
+      description: 'Registry of Open Data on AWS National Oceanographic & Atmospheric Administration National Centers for Environmental Information'
+      roles:
+        - producer
+        - licensor
+      url: s3://noaa-cdr-sea-surface-temp-whoi-pds/
+  license: 'Open Data'
+maintainers:
+  - name: 'Kathryn Berger'
+    orcid: '0000-0001-9731-6519'
+    github: kathrynberger
+bakery:
+  id: 'devseed.bakery.development.aws.us-west-2' # must come from a valid list of bakeries
+  target: pangeo-forge-aws-bakery-flowcachebucketdasktest4-10neo67y7a924
+  resources:
+    memory: 4096
+    cpu: 1024
```

```suggestion
  id: 'pangeo-ldeo-nsf-earthcube'
```

Thanks for this contribution, @kathrynberger!

This PR is not synchronizing with Pangeo Forge Cloud because the bakery you've specified is not one of our currently active bakeries (there is actually only one, at the moment): https://pangeo-forge.org/dashboard/bakeries.

Accepting this suggestion should resolve the failed CI check.

---

Thanks @cisaacstern I realized this after submitting yesterday, thanks for the revision and addition of working bakeries. I've incorporated the change suggested 👍